service/worker/systemconfig/activities.go

```diff
@@ -0,0 +1,64 @@
+package systemconfig
+
+import (
+	"context"
+	"fmt"
+
+	"go.temporal.io/server/common/persistence"
+)
+
+type (
+	UpdateFailoverVersionIncrementInput struct {
+		CurrentFVI int64
+		NewFVI     int64
+	}
+
+	UpdateFailoverVersionIncrementOuput struct {
+		CurrentFVI int64
+		NewFVI     int64
+	}
+
+	activities struct {
+		currentClusterName     string
+		clusterMetadataManager persistence.ClusterMetadataManager
+	}
+)
+
+func (a *activities) UpdateFVI(
```

Comment:
do we need to restart history and other service while doing the update?

Reply:
yes